packages/react-components/react-button/stories/Button/ButtonShape.stories.tsx

```diff
@@ -24,5 +24,29 @@ Shape.parameters = {
     description: {
       story: 'A button can be rounded, circular, or square.',
     },
+    source: {
+      code: `
+import * as React from 'react';
+import { makeStyles, Button } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  wrapper: {
+    columnGap: '15px',
+    display: 'flex',
+  },
+});
+
+export const Shape = () => {
```

Comment:
this duplicity will probably lead into mistakes that might have negative effect on users. Unless this is done automatically behind the scenes by tooling IMHO this is a solution we should not go with.